kedro/framework/project/__init__.py

```diff
@@ -216,10 +216,23 @@ def configure(self, pipelines_module: str | None = None) -> None:
 class _ProjectLogging(UserDict):
     def __init__(self) -> None:
         """Initialise project logging. The path to logging configuration is given in
-        environment variable KEDRO_LOGGING_CONFIG (defaults to default_logging.yml)."""
-        path = os.environ.get(
-            "KEDRO_LOGGING_CONFIG", Path(__file__).parent / "default_logging.yml"
-        )
+        environment variable KEDRO_LOGGING_CONFIG (defaults to conf/logging.yml)."""
+
+        # Check if a user path is set in the environment variable
+        user_logging_path = os.environ.get("KEDRO_LOGGING_CONFIG")
+
+        # Check if the default logging configuration exists
+        default_logging_path = Path("conf/logging.yml")
```

Comment:
Have you considered the CONF_SOURCE or is it only possible to support the happy path?

If this is the case, we should at least update the documentation to reflect that.

Reply:
I will try adding support for custom conf_source.

Reply:
I've been trying out dynamically supporting `CONF_SOURCE` for logging but ran into some challenges. The main issues are related to the timing of loading configuration. So, as you suggested I will update our documentation to reflect this.